src/osemgrep/language_server/LS_e2e.ml

```diff
@@ -0,0 +1,891 @@
+(* Brandon Wu
```

Comment:
maybe put this file under a tests/ subdir, or at least add _test_ in its name, so better match what we do for our other tests.